src/Compilers/CSharp/Test/Semantic/Semantics/TargetTypedNewTests.cs

```diff
@@ -0,0 +1,884 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
+{
```

Comment:
Note: all features get a test plan, which is used to collect test ideas and track their validation. We use this list to decide when to merge a feature from its feature branch into a shared branch (typically `master`).
I started a test plan for "target-typed new": [https://github.com/dotnet/roslyn/issues/28489](https://github.com/dotnet/roslyn/issues/28489)
Since you won't be able to edit the main list, I would suggest that you add a comment with your own additional list there (you'll be able to update it).